exchanges/binance/binance_wrapper.go

```diff
@@ -742,16 +742,21 @@ func (b *Binance) UpdateAccountInfo(ctx context.Context, assetType asset.Item) (
 		if err != nil {
 			return info, err
 		}
-		var currencyDetails []account.Balance
+		accountCurrencyDetails := make(map[string][]account.Balance)
```

Comment:
Could you also do this for case USDTMarginedFutures too?

Reply:
Hi, the case added with USDTMarginatedFutures, is it also available for CoinMarginatedFutures?

but for coin the model is below which does not include account alias:
`// FuturesAccountInformation stores account information for futures account
type FuturesAccountInformation struct {
	Assets []struct {
		Asset                  string  `json:"asset"`
		WalletBalance          float64 `json:"walletBalance,string"`
		UnrealizedProfit       float64 `json:"unrealizedProfit,string"`
		MarginBalance          float64 `json:"marginBalance,string"`
		MaintMargin            float64 `json:"maintMargin,string"`
		InitialMargin          float64 `json:"initialMargin,string"`
		PositionInitialMargin  float64 `json:"positionInitialMargin,string"`
		OpenOrderInitialMargin float64 `json:"openOrderInitialMargin,string"`
		MaxWithdrawAmount      float64 `json:"maxWithdrawAmount,string"`
		CrossWalletBalance     float64 `json:"crossWalletBalance,string"`
		CrossUnPNL             float64 `json:"crossUnPnl,string"`
		AvailableBalance       float64 `json:"availableBalance,string"`
	} `json:"assets"`
	Positions   []FuturesAccountInformationPositions `json:"positions"`
	CanDeposit  bool                                 `json:"canDeposit"`
	CanTrade    bool                                 `json:"canTrade"`
	CanWithdraw bool                                 `json:"canWithdraw"`
	FeeTier     int64                                `json:"feeTier"`
	UpdateTime  time.Time                            `json:"updateTime"`
}`

Reply:
Apologies! I said the wrong asset type.
I meant to do the following for CoinMarginedFutures
```
case asset.CoinMarginedFutures:
		accData, err := b.GetFuturesAccountBalance(ctx)
		if err != nil {
			return info, err
		}
		accountCurrencyDetails := make(map[string][]account.Balance)
		for i := range accData {
			currencyDetails := accountCurrencyDetails[accData[i].AccountAlias]
			accountCurrencyDetails[accData[i].AccountAlias] = append(
				currencyDetails, account.Balance{
					CurrencyName: currency.NewCode(accData[i].Asset),
					TotalValue:   accData[i].Balance,
					Hold:         accData[i].Balance - accData[i].AvailableBalance,
				},
			)
		}

		if info.Accounts, err = account.CollectBalances(accountCurrencyDetails, assetType); err != nil {
			return account.Holdings{}, err
		}
```